include/locales/en-US.h

```diff
@@ -11,6 +11,7 @@
 #define LANG_MANUALLY "Manually"
 #define LANG_CANCEL "Cancel"
 #define LANG_SAVE "Save"
+#define LANG_NTP_QUEUED "NTP Updating"
```

Comment:
Setup is required for all languages that currently exist.

Reply:
ok, unless there is another way to do this, I would be utilizing Google translate for all languages other then english and french.